static/js/impala/formset.js

```diff
@@ -193,13 +212,13 @@ $.zAutoFormset = function(o) {
                     added();
                 }
             }
-        }).data('autocomplete')._renderItem = function(ul, item) {
-            if (!findItem(item).visible) {
-                var $a = $(format('<a><img src="{0}" alt="">{1}</a>',
-                                  [item.icons['32'], item.name]));
-                return $('<li>').data('item.autocomplete', item)
-                                .append($a).appendTo(ul);
-            }
+        }).data('uiAutocomplete')._renderMenu = function(ul, items) {
+            // overwrite _renderMenu to patch in our custom `_renderItemData`
+            // and `_renderItem` to allow for our custom list-filter
```

Comment:
We write our comments as full sentences, so they need to start with an uppercase, and end with a `.`. Just nitpicking ;)